charts/cp-schema-registry/templates/schema-registry-ui.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  {{- if .Values.ui.annotations }}
+  annotations: {{ toYaml .Values.ui.annotations | nindent 4 }}
+  {{- end}}
+  labels:
+    app: {{ template "cp-schema-registry.fullname" . }}
+  {{- if .Values.ui.labels }}
+  {{ toYaml .Values.ui.labels | indent 4}}
+  {{- end}}
+  name: {{ template "cp-schema-registry.fullname" . }}-ui
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "cp-schema-registry.fullname" . }}
+      service: schema-registry-ui
+  template:
+    metadata:
+      annotations:
+      {{- if .Values.ui.annotations }}
+      {{ toYaml .Values.ui.annotations | indent 6 }}
+      {{ end }}
+      labels:
+        app: {{ template "cp-schema-registry.fullname" . }}
+        service: schema-registry-ui
+      {{- if .Values.ui.labels }}
+      {{ toYaml .Values.ui.labels | indent 8}}
+      {{- end}}
+    spec:
+      securityContext:
```

Comment:
For the main Deployement (the registry itself, not the ui), you have put the values into `.Values.securityContext`.
You should probably harmonize those (either put the 2 in values, or the 2 inside the templates/***.yaml directly)

Reply:
Indeed I've put the security context in the values for the deployment itself because I want to push it upstream.
But for there is no UI upstream, I guess I can use the same from the values in the UI security context

Reply:
I don't understand `I guess I can use the same from the values in the UI security context`.

But based on this slack discussion https://blablacar.slack.com/archives/G4G382620/p1618992062099700, we said that we should always put the securityContext into values, so that it's customizable "if needed" (not very likely, but we never know).
So here, UI upstream or not, it doesn't matter I think (or it simplify the decision: we can do what we want)

Reply:
I agree that it's better to use the securitycontext from the values within the UI spec, that's what I just pushed
My sentence was not clear enough I think

Reply:
In your recent patch, you're using `.Values.securityContext`, so the same values for the schema-registry (the daemon itself) and the UI.
I don't think it's appropriate, we might want different uid.

So, you should use a structure like `.Values.ui.securityContext`

Reply:
As we discussed this together, since atm we are using the same values, do you think we could add this feature later on if we gonna need it, honestly I don't think we will IMHO

Reply:
I was maybe thinking too much in a "ideally mode".
So, yeah, let's not spend too much time on this, and keep your current version.